server/app/services/cloud/azure/LocalAzureBlobStorageClient.java

```diff
@@ -0,0 +1,77 @@
+package services.cloud.azure;
+
+import static com.google.common.base.Preconditions.checkNotNull;
+
+import com.azure.storage.blob.BlobClient;
+import com.azure.storage.blob.BlobContainerClient;
+import com.azure.storage.blob.BlobServiceClient;
+import com.azure.storage.blob.BlobServiceClientBuilder;
+import com.azure.storage.blob.sas.BlobSasPermission;
+import com.azure.storage.blob.sas.BlobServiceSasSignatureValues;
+import com.azure.storage.common.sas.SasProtocol;
+import com.typesafe.config.Config;
+import java.net.URLConnection;
+import java.time.Duration;
+import java.time.OffsetDateTime;
+import java.time.ZoneId;
+import java.util.Optional;
+
+/**
+ * Class to use for Azure blob storage in a dev environment.
+ *
+ * <p>This client communicates with the Azurite simulator.
+ */
+class LocalAzureBlobStorageClient extends BaseAzureBlobStorageClient {
```

Comment:
**suggestion:** Gwen had a good point of using `Dev`, since that matches the bin/run-dev naming convention. Would you mind making that update.

Reply:
done